<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>EventRecipesDrugstore</class>
 <widget class="QDialog" name="EventRecipesDrugstore">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>594</width>
    <height>378</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="1" column="0" colspan="2">
    <widget class="CInDocTableView" name="tblItems"/>
   </item>
   <item row="3" column="0">
    <widget class="QPushButton" name="btnSync">
     <property name="text">
      <string>Обновить информацию из аптек</string>
     </property>
    </widget>
   </item>
   <item row="3" column="1">
    <widget class="QPushButton" name="btnClose">
     <property name="text">
      <string>Закрыть</string>
     </property>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QLineEdit" name="edtName"/>
   </item>
   <item row="0" column="1">
    <widget class="QPushButton" name="btnSearch">
     <property name="text">
      <string>Поиск</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="lblLastSync">
     <property name="text">
      <string>Дата и время последней синхронизации:</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>CInDocTableView</class>
   <extends>QTableView</extends>
   <header>library/InDocTable</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
